indico/modules/events/registration/templates/management/regform_invitations.html

```diff
@@ -0,0 +1,72 @@
+{% extends 'events/registration/management/_regform_base.html' %}
+{% from 'events/registration/management/_invitation_list.html' import render_invitation_list %}
+
+{% block subtitle %}
+    {% trans title=regform.title -%}
+        Invitations for "{{ title }}"
+    {%- endtrans %}
+{% endblock %}
+
+{% block content %}
+    {% if regform.is_active %}
+        <div class="action-box">
+            <div class="section">
+                <div class="icon icon-users"></div>
+                <div class="text">
+                    <div class="label">
+                        {% trans %}Invite people{% endtrans %}
+                    </div>
+                    <div>
+                        {% trans %}Invite people to register for your event.{% endtrans %}
+                    </div>
+                </div>
+                <div class="toolbar right">
+                    <a class="i-button icon-user arrow js-dropdown" data-toggle="dropdown">Invite</a>
+                    <ul class="dropdown">
+                        <li>
+                            <a class="js-invite-user" data-href="{{ url_for('.invite', regform) }}"
+                               data-title="{% trans %}Invite user to register{% endtrans %}">
+                                {% trans %}New user{% endtrans %}
+                            </a>
+                        </li>
+                        <li>
+                            <a class="js-invite-user" data-href="{{ url_for('.invite', regform, existing=True) }}"
+                               data-title="{% trans %}Invite user to register{% endtrans %}">
+                                {% trans %}Indico users{% endtrans %}
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    {% endif %}
+
+    <div class="i-box invitation-list" id="invitation-list">
+        {{ render_invitation_list(invitations) }}
+    </div>
+
+    <script>
+        $('#invitation-list').on('indico:confirmed', '.js-delete-invitation', function(evt) {
+            evt.preventDefault();
+
+            var $this = $(this);
+            $.ajax({
+                url: $this.data('href'),
+                method: $this.data('method'),
+                complete: IndicoUI.Dialogs.Util.progress(),
+                error: handleAjaxError,
+                success: function() {
+                    $this.closest('tr').remove();
+                }
+            });
+        });
+
+        $('.js-invite-user').ajaxDialog({
+            onClose: function(data) {
+                if (data) {
+                    $('#invitation-list').html(data.invitation_list);
```

Comment:
no, camelize is just a workaround because we didn't want to update all the legacy-ish angular code. it's much nicer if you can search e.g. for `invitation_list` and get both js and related python results